src/Jobs/Recruit.Vacancies.Jobs/Program.cs

```diff
@@ -76,8 +77,16 @@ public static async Task Main(string[] args)
             var host = builder.Build();
             using (host)
             {
+                CheckInfrastructure(host.Services);
```

Comment:
I don't think we'll get any logging of the exception would we?

Reply:
I'll put back the "The Job has met with a horrible end!!". I miss it.